### DISCUSSION TITLE
rt3s digital channels missing squelch setting.

### DISCUSSION THREAD

**Discussion by gerald-clark** [hmatuschek/qdmr#500] [Q&A]
When editing digital channels for my RT3S  there is no setting for squelch.
I had to resort to the retevis supplied CPE to change squelch from 0.

**Comment by hmatuschek** [comment]
You can actually already set it. The DMR squelch is a bug (in my opinion) of some  TyT/Retevis devices. It makes no sense to set a DMR squelch. However, you may need to set it using the TyT channel extension:

https://dm3mat.darc.de/qdmr/manual/ch04s03.html#extTyTChannelSettings

Enable *device specific settings* in the settings dialog, and add an TyT extension to the channels. If you read the codeplug from the device, there should already be one. Set the `dmrSquelch` property there.

![grafik](https://github.com/user-attachments/assets/02033e18-91c6-466d-a13c-4ae8cc5b0fd3)

**Reply by gerald-clark** [reply]
Thanks.  My reason for needing this is broken squelch during scan when default squelch level was '0'.
Scanning is working fine now.
